README.md

```diff
@@ -1,4 +1,4 @@
 # first-repo
 repository of GitHub file for the VeroSkills coding class
 additions have been made
-more additions have been made
\ No newline at end of file
+miore additions have been made
```

Comment:
should say "more changes have been made" instead of "miore  changes have  been made"